code/.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
```

Comment:
Great job this week Evelina 🌟 you should be proud of yourself! You have clean code that is easy to follow. And I am happy to hear you lowered the demands on yourself this week because you have done a great you and you need to be proud and happy over what you have accomplished during this short period of time! 💟